tools/go.mod

```diff
@@ -1,6 +1,6 @@
 module tools/tools
 
-go 1.17
+go 1.18
 
 require github.com/golangci/golangci-lint v1.31.0
```

Comment:
Out of PR scope comment

Wow

This one should be upgraded somehow.

Or maybe this pre go tool way is not used?

Go 1.18 is not Go 1.24, so go mod -tool is not available, also golangci-lint installed this way is discouraged.